Templates/ServiceBusQueueTrigger-CSharp/run.csx

```diff
@@ -1,7 +1,7 @@
 using System;
 using System.Threading.Tasks;
 
-public static void Run(string myQueueItem, TraceWriter log)
+public static void Run(string mySbMsg, TraceWriter log)
 {
-    log.Verbose($"C# ServiceBus queue trigger function processed message: {myQueueItem}");
+    log.Verbose($"C# ServiceBus queue trigger function processed message: {mySbMsg}");
```

Comment:
change your log statements to Info level rather than Verbose (see my open PR on this repo)